.github/workflows/ci.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.10', '3.9', '3.8', '3.7']
+        python-version: ['3.10', '3.9', '3.8']
```

Comment:
Lots of changes where the cookiecutter is removing Python 3.7 because I removed it from `cookiecutter.json`.